README.md

```diff
@@ -1,17 +1,15 @@
-# conda-remove-env
+# conda_remove_env
 
 Remove one or more conda environments
 
-
 1. Clone the repo
 
 ```
-git clone https://github.com/trevor-moon/conda-remove-envs.git
```

Comment:
This line can remain unchanged - the link will be broken since the url uses the repo name `conda-remove-envs`

Reply:
PR Updated